File: javascript/gemini/p10.js
const fs = require("fs");

function getValueFromArray(arr, size, index) {
  if (index < 0 || index >= size) {
    return -1;
  }
  return arr[index];
}

let id_sequence = [100, 200, 300, 400];
// Reading from stdin
let input = parseInt(fs.readFileSync(0).toString());

let result = getValueFromArray(id_sequence, id_sequence.length, input);
process.exit(result);

File: python/gemini/p10.py
import sys


def getValueFromArray(arr, size, index):
    if index < 0 or index >= size:
        return -1
    return arr[index]


id_sequence = [10, 20, 30, 40, 50]

try:
    user_input = int(input())
    result = getValueFromArray(id_sequence, len(id_sequence), user_input)
    sys.exit(result)
except EOFError:
    pass
